packages/grafana-ui/src/components/Table/Table.test.tsx

```diff
@@ -144,10 +144,10 @@ describe('Table', () => {
       const rows = within(getTable()).getAllByRole('row');
       expect(rows).toHaveLength(5);
       expect(getRowsData(rows)).toEqual([
-        { time: '2021-01-01 00:00:00', temperature: '10', link: '10' },
-        { time: '2021-01-01 03:00:00', temperature: 'NaN', link: 'NaN' },
-        { time: '2021-01-01 01:00:00', temperature: '11', link: '11' },
-        { time: '2021-01-01 02:00:00', temperature: '12', link: '12' },
+        { time: '2021-01-01 00:00:00', temperature: '10', link: '${__value.text} interpolation' },
+        { time: '2021-01-01 03:00:00', temperature: 'NaN', link: '${__value.text} interpolation' },
+        { time: '2021-01-01 01:00:00', temperature: '11', link: '${__value.text} interpolation' },
+        { time: '2021-01-01 02:00:00', temperature: '12', link: '${__value.text} interpolation' },
```

Comment:
@dprokop no, but since this function is in grafana/ui and does not have access to a real templateSrv replace function, the replaceVariables call is just mocked in this test, so this is there return value of the mocked templateSrv replace function